java-checks-test-sources/default/src/main/java/checks/S3252_StaticMemberAccessCheckSample/S3252TestDefaultPackage.java

```diff
@@ -0,0 +1,14 @@
+public class S3252TestDefaultPackage {
```

Comment:
Can we rename this class in a way that does not feel like this class is a package? Something like `ClassInDefaultPackage`? The package is already in the folder name that contains the class.

Reply:
:+1: